exts/cesium.omniverse/cesium/omniverse/ui/attributes/custom_attribute_widgets.py

```diff
@@ -105,3 +106,18 @@ def custom_slider(stage, attr_name, metadata, property_type, prim_paths, *args,
         )
 
     return custom_slider
+
+
+def build_common_raster_overlay_properties():
```

Comment:
It's starting to feel like there are too many groups with just a single option. Can they be consolidated? I guess copy whatever Unreal is doing.

![image](https://github.com/CesiumGS/cesium-omniverse/assets/915398/42140095-1e90-437d-bdff-20c7bbac4db7)

Reply:
I agree. I took a stab at this maybe two months ago, but didn't find a huge improvement. Here's what I found. Looking again, I think we can improve the UI but can't match Unreal. 

Here's a comparison of the ion raster overlay UIs across frameworks. The OV interface has been tweaked to what I think is best:

![image](https://github.com/CesiumGS/cesium-omniverse/assets/1040194/0a552830-465d-46cd-84fe-0b9234d2125e)

The core limitation is that in Omniverse, everything has to be in a collapsible group. (At least I haven't found a way around this.) If you don't put a `LayoutProperty` in a `LayoutGroup`, OV will do it for you. In the image above, OV is providing "Extra Properties." We can replace that default string, but can't de-group it like in Unreal.

Looking back, I think "Extra Properties" is better than each isolated `LayoutProperty` having it's own redundant `LayoutGroup`. It's possible there's a way to customize `CustomLayoutFrame`, but I'm not seeing any easy solution.